quickannotator/api/v1/annotation/__init__.py

```diff
@@ -176,6 +184,25 @@ def get(self, args, image_id, annotation_class_id):
             result = db_session.execute(stmt).fetchall()
             return result, 200
         
+@bp.route('/<int:image_id>/<int:annotation_class_id>/bulk')
+class BulkAnnotation(MethodView):
+    @bp.arguments(PostAnnArgsSchema(many=True), location='json')
+    # @bp.response(200, AnnRespSchema(many=True))
+    def post(self, args, image_id, annotation_class_id):
+        """     post multiple new annotations to the db.
+        
+        This method is primarily used for ground truth annotations. Predictions should only be saved by the model.
+        """
+        isgt = True
+        polygons = [shape(annotation['polygon']) for annotation in args]
+
+        annotations = bulk_insert_annotations(image_id, annotation_class_id, isgt, polygons, from_work_mag=False)
+        tileids = [ann.tile_id for ann in annotations]
+        bulk_upsert_tiles(annotation_class_id, image_id, tileids, hasgt=isgt)
```

Comment:
i can't see from this review scope -- what is the purpose of this? i see insert and then and an upsert? what functionlaity is not completed in the first insert?

Reply:
OoOO i see - one is for annotations and one is for tiles - sorry missed that